xapian-letor/configure.ac

```diff
@@ -429,15 +429,6 @@ if test -n "${EATMYDATA-unset}" ; then
   AC_PATH_PROG([EATMYDATA], [eatmydata], [])
 fi
 
-dnl Used by tests/harness/testsuite.cc
-AC_CHECK_FUNCS([sigaction])
-AC_MSG_CHECKING([for sigsetjmp and siglongjmp])
-AC_TRY_COMPILE([#include <setjmp.h>],
-  [sigjmp_buf jb; if (sigsetjmp(jb, 1)) { siglongjmp(jb, 1);  }],
-  AC_DEFINE([HAVE_SIGSETJMP], [1], [Define to 1 if you have the 'sigsetjmp' function])
-  AC_MSG_RESULT([yes]),
-  AC_MSG_RESULT([no]))
-
```

Comment:
Configure is defining some backend build macros, removing which is causing the harness to fail to build. Cutting out "Check which database backends should be built." portion would require code changes in the harness, which is symlinked right now.